README.md

```diff
@@ -11,28 +11,58 @@ Farcaster is a protocol for building decentralized social applications. If you w
 
 ## Table of Contents
 
-1. [Introduction](#1-introduction)
-   1. [Prior Art](#11-prior-art)
-   2. [Proposal](#12-proposal)
-2. [Identity](#2-identity)
-   1. [Farcaster ID's](#21-farcaster-ids)
-   2. [Farcaster Names ](#22-farcaster-names)
-   3. [Recovery](#23-recovery)
-3. [Delta Graph](#3-delta-graph)
-   1. [Synchronization](#31-synchronization)
-   2. [Ordering](#32-ordering)
-   3. [Authentication](#33-authentication)
-   4. [Bounding Graph Size](#34-bounding-graph-size)
-4. [Hubs](#4-hubs)
-   1. [Synchronization](#41-synchronization)
-5. [Applications](#5-applications)
-6. [Upgradeability](#6-upgradeability)
-   1. [Release Schedule](#61-release-schedule)
-7. [Acknowledgements](#7-acknowledgements)
-8. [Appendix A: Proposals](#8-appendix-a-proposals)
-9. [Appendix B: Delta Specifications](#9-appendix-b-delta-specifications)
-10. [Appendix C: Application Specifications](#10-appendix-c-application-specifications)
-11. [Appendix D: Contract Specifications](#11-appendix-d-contract-specifications)
+- [Farcaster Protocol](#farcaster-protocol)
```

Comment:
No idea why my linter expanded the TOC like this.....lmk if you want me to change it

Reply:
yes please, i intentionally kept the toc small